src/background.js

```diff
@@ -4,6 +4,7 @@
 
 window.scrNoti = window.scrNoti || {};
 const seenMessages = {};
+let port = null;
```

Comment:
I would prefere the name `nativeConnection` to `port` I think.

Reply:
Done in last commit.